commands/FBDebugCommands.py

```diff
@@ -7,6 +7,8 @@
 import sys
 import os
 import re
+# Can be removed when Python 2 support is removed.
+from __future__ import print_function
```

Comment:
I thought this would raise an exception under python 3, but it doesn't so that's good.

Reply:
I picked up this branch to run with Xcode 11 today (thank you @liuliu!) and got an error that this `__future__ import` needs to be at the top of the file. Moving it above the other imports silenced the warning.